.github/workflows/tembo_ai.yaml

```diff
@@ -0,0 +1,62 @@
+name: Tembo AI Gateway Workflow
+
+permissions:
+  pull-requests: write
+  deployments: write
+  id-token: write
+  contents: read
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./inference-gateway/
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+    - '.github/workflows/tembo_ai.yaml'
+    - 'inference-gateway/**'
+  push:
+    branches:
+      - main
+    paths:
+    - '.github/workflows/tembo_ai.yaml'
+    - 'inference-gateway/**'
+
+jobs:
+  tests:
+    name: Run tests
+    runs-on:
+      - self-hosted
+      - dind
+      - xlarge-16x16
```

Comment:
is 16x16 needed? If not can pick smaller with "small" or "large-8x8". Also should include label "amd64"

Reply:
Thanks, should be able to run this on something smaller. Where can I full all the enum values for these labels?